Library/Homebrew/utils/link.rb

```diff
@@ -0,0 +1,68 @@
+module Link
```

Comment:
Cool with me. I wasn't sure about the idiomatic ruby way on this was and the modules under `/utils` seemed to follow both approaches.